src/Components/Web.JS/src/GlobalExports.ts

```diff
@@ -104,6 +107,8 @@ export const Blazor: IBlazor = {
     NavigationLock,
     getJSDataStreamChunk: getNextChunk,
     attachWebRendererInterop,
+    performProgrammaticEnhancedNavigation,
```

Comment:
The plan is to consume these APIs in the hot reload scripts [here](https://github.com/dotnet/sdk/blob/62d3e897cce38a3df8d0ff4eb489a3f2e16f6dce/src/BuiltInTools/BrowserRefresh/WebSocketScriptInjection.js#L193) to trigger a refresh for statically rendered page.